src/terra-dev-site/app/components.2/NotificationBanner.app.mdx

```diff
@@ -0,0 +1,74 @@
+import PropsTable from './NotificationBannerPropsTable?dev-site-props-table';
```

Comment:
Why can't you import the props from the notification banner component? Because it returns null?

Reply:
yes because of null

Reply:
react doc gen is an imperfect tool for this use case 😕